3-UsingAPIs/Week2/homework/ex4-diceRace.js

```diff
@@ -15,18 +15,23 @@ const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
   const dice = [1, 2, 3, 4, 5];
-  // TODO complete this function; use Promise.race() and rollDie()
+   const promises = dice.map(rollDie);
+   return Promise.race(promises);
```

Comment:
Remember to only assign things to variables if they are either very complex assignments or are used multiple times in your function

```suggestion
   return Promise.race(dice.map(rollDie));
```